src/server/models/listItem/listItem.ts

```diff
@@ -324,22 +317,24 @@ type Nullable<T> = T | undefined | null;
 /**
  * updates and PUBLISHES!
  */
-export async function update(
-  id: ListItem["id"],
-  userId: User["id"],
-  legacyDataParameter?: DeserialisedWebhookData
-): Promise<void> {
-  logger.info(`user ${userId} is attempting to update ${id}`);
+export async function update(id: ListItem["id"], userId: User["id"], legacyDataParameter?: DeserialisedWebhookData) {
+  const updateLogger = logger.child({ listItemId: id, user: userId, method: "ListItem.update" });
```

Comment:
logs in this format: 
`info $the_message {"listItemId": 1, "userId": 2, "method": "listItem.update", "service":"server","timestamp":"2023-02-02T15:38:28.824Z"}`

service: server and timestamp are inherited from the parent logger